mods/gen5/formats-data.js

```diff
@@ -3130,4 +3134,94 @@ exports.BattleFormatsData = {
 		eventOnly: true,
 		requiredItem: "Shock Drive",
 	},
+	ufo: {
+		inherit: true,
+		isNonStandard: false,
```

Comment:
Good point. We'd need to make an exception for them for Hackmons. But I'd rather code the exception at the Hackmons level somehow.